packages/server/lib/browsers/electron.coffee

```diff
@@ -148,6 +142,23 @@ module.exports = {
       debug("debugger attached")
     catch err
       debug("debugger attached failed %o", { err })
+      throw err
+
+    originalSendCommand = webContents.debugger.sendCommand
```

Comment:
You could use [_.wrap](https://lodash.com/docs/4.17.15#wrap) here. 

```coffeescript
webContents.debugger.sendCommand = _.wrap(webContents.debugger.sendCommand, (originalSendCommand, message, data) -> 
```

Reply:
I kinda prefer doing it without lodash, it's only 1 more line of code and it doesn't require the reader to know the function signature of `_.wrap` or how `_.wrap` binds the wrapped function.